Lib/test/test_clinic.py

```diff
@@ -2197,6 +2197,57 @@ class Foo "" ""
                 expected_error = err_template.format(invalid_kind)
                 self.expect_failure(block, expected_error, lineno=3)
 
+    def test_invalid_getset(self):
+        annotations = ["@getter", "@setter"]
+        for annotation in annotations:
+            with self.subTest(annotation=annotation):
+                block = f"""
+                    module foo
+                    class Foo "" ""
+                    {annotation}
+                    Foo.property -> int
+                """
+                expected_error = f"{annotation} method cannot define a return type"
+                self.expect_failure(block, expected_error, lineno=3)
+
+                block = f"""
+                   module foo
+                   class Foo "" ""
+                   {annotation}
+                   Foo.property
+                       obj: int
+                       /
+                """
+                expected_error = f"{annotation} method cannot define parameters"
```

Comment:
Doesn't look like the `expected_error` variable here is actually used; did you mean to do this? :)

```suggestion
                expected_error = f"{annotation} method cannot define parameters"
                self.expect_failure(block, expected_error)
```

Reply:
Thank you for catching it!